README.md

```diff
@@ -330,6 +370,40 @@ Post.search do
 end
 ```
 
+#### Nested Documents search with `ChildOf` and `ParentWhich`
```

Comment:
I dont think this naming is good. Lets be more verbose and document this as 'Nested documents search based on childs or parent document properties'